src/commands/status/core.js

```diff
@@ -0,0 +1,152 @@
+const { table } = require('table');
+const fetch = require('node-fetch');
+const chalk = require('chalk');
+
+const BaseCommand = require('../../oclif/command/BaseCommand');
+const CoreService = require('../../core/CoreService');
+
+const ContainerIsNotPresentError = require('../../docker/errors/ContainerIsNotPresentError');
+
+class CoreStatusCommand extends BaseCommand {
+  /**
+   * @param {Object} args
+   * @param {Object} flags
+   * @param {DockerCompose} dockerCompose
+   * @param {CoreService} coreService
+   * @param {Config} config
+   * @return {Promise<void>}
+   */
+  async runWithDependencies(
+    args,
+    flags,
+    dockerCompose,
+    createRpcClient,
+    config,
+  ) {
+    const rows = [];
+
+    const coreService = new CoreService(
+      createRpcClient(
+        {
+          port: config.get('core.rpc.port'),
+          user: config.get('core.rpc.user'),
+          pass: config.get('core.rpc.password'),
+        },
+      ),
+      dockerCompose.docker.getContainer('core'),
+    );
+
+    const insightURLs = {
+      evonet: 'http://insight.evonet.networks.dash.org:3001/insight-api',
+      testnet: 'https://testnet-insight.dashevo.org/insight-api',
+      mainnet: 'https://insight.dash.org/insight-api',
+    };
+
+    // Collect data
+    const { result: blockchainInfo } = await coreService.getRpcClient().getBlockchainInfo();
+    const { result: networkInfo } = await coreService.getRpcClient().getNetworkInfo();
+    const { result: mnsyncStatus } = await coreService.getRpcClient().mnsync('status');
+    const { result: peerInfo } = await coreService.getRpcClient().getPeerInfo();
+    const latestVersionRes = await fetch('https://api.github.com/repos/dashpay/dash/releases/latest');
+    const latestVersion = await latestVersionRes.json();
+    const insightBlockHeightRes = await fetch(`${insightURLs[config.options.network]}/status`);
```

Comment:
For local network, it will be invalid url. Let’s skip it if the `!insightURL[..`